lib/modules/widgets/widgets/recent_episodes.php

```diff
@@ -0,0 +1,106 @@
+<?php
+namespace Podlove\Modules\Widgets\Widgets;
+
+use \Podlove\Model\Podcast;
+
+class RecentEpisodes extends \WP_Widget {
+
+	public function __construct() {
+		parent::__construct(
+			'podlove_recent_episodes_widget',
+			'Podlove Recent Episodes',
+			array( 'description' => __( 'Shows the recent episodes of your podcast.', 'podlove' ) )
+		);
+	}
+
+	private static function get_duration_text_string($episode) {
+		$duration = new \Podlove\Duration( $episode->duration );
+		$duration_string = '';
+
+		if ( $duration->hours > 1 ) {
+			$duration_string .= $duration->hours . __(' hours ', 'podlove');
+		} elseif ( $duration->hours = 1 ) {
+			$duration_string .= $duration->hours . __(' hour ', 'podlove');
+		}
+
+		if ( $duration->minutes >= 1 )
+			$duration_string .= $duration->minutes . __(' minutes ', 'podlove');
+
+		if ( $duration->hours == 0 && $duration->minutes == 0 )
+			$duration_string .= $duration->seconds . __(' seconds', 'podlove');
+
+		return $duration_string;
```

Comment:
Wäre es nicht sinnvoll, die Logik in die Duration-Klasse auszulagern? z.B. `$duration->get('human readable')`